controllers/cloud.redhat.com/providers/deployment/impl.go

```diff
@@ -99,17 +97,13 @@ func initDeployment(app *crd.ClowdApp, env *crd.ClowdEnvironment, d *apps.Deploy
 	}
 
 	c := core.Container{
-		Name:         nn.Name,
-		Image:        pod.Image,
-		Command:      pod.Command,
-		Args:         pod.Args,
-		Env:          envvar,
-		Resources:    ProcessResources(&pod, env),
-		VolumeMounts: pod.VolumeMounts,
-		Ports: []core.ContainerPort{{
```

Comment:
This is now factored out into the metrics provider, correct?

Reply:
Correct - you can see it there right?

Reply:
This was left over from the other half of the refactor I think

Reply:
After double-checking, it is indeed there.